plugin.yaml

```diff
@@ -46,3 +92,90 @@ node_types:
+        description: >
+          User used to login to BIG IP GTM instance
+      password:
+        type: string
+        description: >
+          Password used to login to BIG IP GTM instance
+
+  cloudify.nodes.bigip.gtm.DNSLoadBalancer:
+    derived_from: cloudify.nodes.Root
+    properties:
+      partition:
+        description: *partition_desc
+        type: string
+        required: true
+      ltm_server:
+        description: *ltm_server_desc
+        type: string
+        required: true
+      record_type:
+        description: *record_type_desc
+        type: string
+        required: true
+      virtual_server_config:
+        description: >
+          Configuration key-value data to be passed as-is in order to create
+           the virtual server resource, this require checking the F5 GTM
+           Documentation in order to pass the payload needed for the virtual server
+        type: cloudify.datatypes.bigip.gtm.VirtualServer
+        required: true
+      pool_config:
+        description: >
+          Configuration key-value data to be passed as-is in order to create
+           the GTM server pool, this require checking the F5 GTM
+           Documentation in order to pass the payload needed for the pool
+        type: cloudify.datatypes.bigip.gtm.Pool
+        required: true
+      pool_member_config:
+        description: >
+          Configuration key-value data to be passed as-is in order to add
+           the virtual server to pool, this require checking the F5 GTM
+           Documentation in order to pass the payload needed for the pool
+           member
+        type: cloudify.datatypes.bigip.gtm.PoolMember
+        required: true
+      wipe_config:
+        description: >
+          Configuration key-value data to be passed as-is in order to create
+           the GTM wipe ip, this require checking the F5 GTM
+           Documentation in order to pass the payload needed for the wipe
+        type: cloudify.datatypes.bigip.gtm.Wipe
+        required: true
+    interfaces:
+      cloudify.interfaces.lifecycle:
+        create:
+          implementation: bigip.bigip_plugin.bigip_gtm.create_dns_loadbalancer
+          inputs:
+           partition:
+             default: { get_property: [SELF, partition]}
+           ltm_server:
+             default: { get_property: [SELF, ltm_server]}
+           record_type:
+             default: { get_property: [SELF, record_type]}
+           virtual_server_config:
+             default: { get_property: [SELF, virtual_server_config]}
+           pool_config:
+             default: { get_property: [SELF, pool_config]}
+           pool_member_config:
+             default: { get_property: [SELF, pool_member_config]}
+           wipe_config:
+             default: { get_property: [SELF, wipe_config]}
+        delete:
+          implementation: bigip.bigip_plugin.bigip_gtm.delete_dns_loadbalancer
+
+relationships:
+
+  cloudify.bigip.gtm.relationships.managed_by_gtm:
```

Comment:
What is the point of having a special relationship if there is no operation mapping?

Reply:
We are planning to get rid of the way we do it in the Kubernetes plugin, because it is not as good as a property

Reply:
Fixed and removed